source/StigData/Processed/SqlServer-2016-Instance-2.9.org.default.xml

```diff
@@ -5,9 +5,9 @@
     Each setting in this file is linked by STIG ID and the valid range is in an
     associated comment.
 -->
-<OrganizationalSettings fullversion="2.7">
+<OrganizationalSettings fullversion="2.9">
   <!-- Ensure SQL authentication logins are populated from organizational settings.-->
   <OrganizationalSetting id="V-213964" Ensure="" Name="" />
   <!-- Ensure 'V-214029' is populated with a non-default SA account name-->
-  <OrganizationalSetting id="V-214029" VariableValue="stig_sa" />
+  <OrganizationalSetting id="V-214029" VariableValue="" />
```

Comment:
Can you update this to have a default value like the n-1 version ? 

"stig_sa" - will be good to merge after this change